.github/workflows/ci.yml

```diff
@@ -56,5 +56,8 @@ jobs:
         with:
           connection-string-name: OracleConnectionString
           tag: NHibernatePersistence
+          registry-login-server: index.docker.io
```

Comment:
Looking at other CI workflows, they haven't added this.

I think we can drop this line since we are using the default.